server_help_test.go

```diff
@@ -69,7 +103,7 @@ func StartTestCluster(t *testing.T, size int, stdout, stderr io.Writer) (*TestCl
 
 	for serverN := 0; serverN < size; serverN++ {
 		srvPath := filepath.Join(tmpPath, fmt.Sprintf("srv%d", serverN+1))
-		requireNoError(t, os.Mkdir(srvPath, 0700), "failed to make server path")
+		requireNoErrorf(t, os.Mkdir(srvPath, 0700), "failed to make server path")
```

Comment:
Can we pluck these changes into a separate diff since they do not change any substance to the code. Just trying to remove lines to review as change vs not

Reply:
Ack makes sense! As i do that, any opposition to using [testify](https://github.com/stretchr/testify) to clean this up further ?

Reply:
We have maintained using only the stdlib as it does have nice properties of not conflicting or dealing with any dependencies. I'm not against it longer term, but for this round of changes lets stick to stdlib. 

I want to keep the scope of change tighter as we get to the meat of the changes

Reply:
chipping away and doing small PRs like errors.Is will also not change behavior and good to clean up this larger PR

Reply:
ACK makes sense, it's not necessary, just a nicety. I can move _some_ of the `errors.Is` stuff but I did change the error signatures of some of these methods, and it does actually break if you don't do `errors.Is` anymore